docusaurus/docs/key-concepts/backend-plugins/index.md

```diff
@@ -11,41 +11,35 @@ keywords:
 sidebar_position: 1.5
 ---
 
-# Backend plugins
+# Plugin backend system
 
-The Grafana plugin system for backend development allows you to integrate Grafana with virtually anything and offer custom visualizations. The system is based on HashiCorp's [Go Plugin System over RPC](https://github.com/hashicorp/go-plugin). Our implementation of the Grafana server launches each backend plugin as a subprocess and communicates with it over [gRPC](https://grpc.io/).
+Introduced in Grafana v7.0, plugins with backend components allow you to integrate Grafana with virtually anything and offer custom visualizations. The plugin backend system is based on HashiCorp's [Go Plugin System over RPC](https://github.com/hashicorp/go-plugin) and supports server-side plugin elements. The Grafana server launches each plugin backend as a subprocess and communicates with it over [gRPC](https://grpc.io/).
 
-This document explains the system's background, use cases, benefits, and key features.
-
-## Background
+:::note
+A plugin with a backend usually requires a frontend component as well. For example, some backend data source plugins need query editor components on the frontend.
```

Comment:
This was not accurate in the current documentation, a frontend component is always necessary.

```suggestion
A plugin with a backend requires a frontend component as well. For example, all data source plugins need a query editor component on the frontend.
```